src/rules/valid-title.test.ts

```diff
@@ -52,10 +52,10 @@ runRuleTester('valid-title', rule, {
       options: [{ disallowedWords: ['descriptive'] }],
     },
     {
-      code: 'test(`that the value is set properly`, function () {})',
+      code: 'test.step(`that the value is set properly`, function () {})',
```

Comment:
Let's leave the existing test and just add new ones. Don't like changing existing tests ever.